src/configs/binary-elementwise-config.c

```diff
@@ -399,6 +399,12 @@ static void init_f32_vadd_config(void) {
     f32_vadd_config.minmax.ropc_ukernel = (xnn_vbinary_ukernel_fn) xnn_f32_vaddc_minmax_ukernel__scalar_u8;
     f32_vadd_config.init.f32_minmax = xnn_init_f32_minmax_scalar_params;
     f32_vadd_config.minmax.element_tile = 8;
+  #elif XNN_ARCH_HEXAGON
```

Comment:
normally we'd configure the kernels when they are ready to put into production.  e.g.  hvx kernels.
since these are placeholders, perhaps change the ppc above to
#elif XNN_ARCH_PPC64 || XNN_ARCH_HEXAGON and then add a hexagon section when there is hexagon specific kernels?

Reply:
I think that's very reasonable. I also see XNN_ARCH_RISCV is configured using scalar as well. Should we put all three cases together for now? So it will become:
#elif XNN_ARC_RISCV || XNN_ARCH_PPC64 || XNN_ARCH_HEXAGON